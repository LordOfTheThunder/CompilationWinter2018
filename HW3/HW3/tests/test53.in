int retInt() {
	int t;
}
bool retBool() {
	bool s;
}
int TestRetInt(int s) {
	int t = retInt();
	s = retInt();
	s = retBool();
	int x;
	x = retInt();
}




void main() { print("I hate this course"); }



