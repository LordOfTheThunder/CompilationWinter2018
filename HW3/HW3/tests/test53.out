---end scope---
t INT 0
---end scope---
s BOOL 0
line 10: type mismatch
